tests/foreman/cli/test_syncplan.py

```diff
@@ -400,11 +397,9 @@ def test_negative_synchronize_custom_product_current_sync_date(self):
             )
             # validate the error message once unstubbed (#3611)
 
-    # This Bugzilla bug is private. It is impossible to fetch info about it.
-    @stubbed('Unstub when BZ1279539 is fixed')
     @tier4
     def test_positive_synchronize_custom_product_current_sync_date(self):
-        """Create a sync plan with current datetime as a sync date, add a
+        """Create a sync plan with a past datetime as a sync date, add a
```

Comment:
Why did you decide that is expected behaviour? Even test case name point into word `current`